error.py

```
@@ -0,0 +1,45 @@
+    print("Received name:", name)  # ❌ print() not allowed
+    print("Received email:", email)
+
+    # ❌ No validation: should check if name/email start with special characters
+    # Expected: check name[0].isalnum() before saving
+
+    user_data = {
+        "name": name,
+        "email": email
+    }
+
+    # ❌ Uses eval() – very risky, disallowed
```

Ruff (0.14.4)

Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)